arelle/plugin/validate/NL/rules/fr_kvk.py

```diff
@@ -82,6 +86,56 @@ def rule_fr_kvk_2_01(
                 )
 
 
+def etree_iter(node: ModelObject | _Element) -> Any:
+    yield node
+    for child in node.iterchildren():
+        for n_d in etree_iter(child):
+            yield n_d
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NT16,
+        DISCLOSURE_SYSTEM_NT17,
+    ],
+)
+def rule_fr_kvk_2_02(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation] | None:
+    """
+    FR-KVK-2.02: The attribute 'xml:lang' MUST contain the same value within an XBRL instance document
+    """
+    modelXbrl = val.modelXbrl
+    docXmlLang = []
+    for fact in modelXbrl.facts:
+        if fact.xmlLang and fact.xmlLang not in docXmlLang:
+            docXmlLang.append(fact.xmlLang)
+    assert modelXbrl.modelDocument is not None
+    if modelXbrl.modelDocument.type in (ModelDocument.Type.INLINEXBRL, ModelDocument.Type.INLINEXBRLDOCUMENTSET, ModelDocument.Type.UnknownXML):
```

Comment:
My reasoning could be off here, but I don't think we need to support inline XBRL here. FR-KVK-1.01 explicitly requires an `.xbrl` file extension and nothing in the KVK rules mentions inline XBRL that I've seen.

Reply:
Aarons assessment is correct. There is no ixbrl with this type of filing at the moment.